packages/lib/modules/pool/PoolList/PoolListProvider.tsx

```diff
@@ -42,6 +64,30 @@ export function _usePoolList({ fixedPoolTypes }: { fixedPoolTypes?: GqlPoolType[
     }
   }, [userAddress])
 
+  useEffect(() => {
+    if (_displayType) {
+      setDisplayType(_displayType)
+    }
+  }, [_displayType])
+
+  useEffect(() => {
+    if (_hideProtocolVersion) {
+      setHideProtocolVersion(_hideProtocolVersion)
+    }
+  }, [_hideProtocolVersion])
+
+  useEffect(() => {
+    if (_hidePoolTypes) {
+      setHidePoolTypes(_hidePoolTypes)
+    }
+  }, [_hidePoolTypes])
+
+  useEffect(() => {
+    if (_hidePoolTags) {
+      setHidePoolTags(_hidePoolTags)
+    }
+  }, [_hidePoolTags])
```

Comment:
Why do we need to do this state syncing? Can we not just use the values that come through as props? There shouldn't be a case where we want to also allow some user interaction to change these values.

Reply:
Changed to just use props with a default.